Formula/prokka.rb

```diff
@@ -0,0 +1,46 @@
+class Prokka < Formula
+  # cite Seemann_2014: "https://doi.org/10.1093/bioinformatics/btu153"
+  desc "Rapid annotation of prokaryotic genomes"
+  homepage "https://github.com/tseemann/prokka"
+  url "https://github.com/tseemann/prokka/archive/v1.13.tar.gz"
+  sha256 "0090acd6e3a8cb2d517fe0933c717c4f8271c60d0a493f7c4927504d77a8db4f"
+
+  depends_on "cpanminus" => :build
+
+  depends_on "aragorn"
+  depends_on "barrnap"
+  depends_on "bioperl"
+  depends_on "blast"
+  depends_on "hmmer"
+  depends_on "infernal"
+  depends_on "minced"
+  depends_on "parallel"
+  depends_on "perl" unless OS.mac?
+  depends_on "prodigal"
+  depends_on "tbl2asn"
+
+  def install
+    # remove all bundled binaries and use brew ones
+    rm_r "binaries"
+    # install CPAN modules
+    ENV.prepend_create_path "PERL5LIB", prefix/"perl5/lib/perl5"
+    system "cpanm", "--self-contained", "-l", prefix/"perl5",
```

Comment:
Try removing `--self-contained`. I don't believe it's necessary, and I had trouble with it. I opened an issue upstream with `cpanminus` https://github.com/miyagawa/cpanminus/issues/563